Sources/Workspace/UserToolchain.swift

```diff
@@ -36,7 +36,7 @@ public final class UserToolchain: Toolchain {
 
     public let extraCCFlags: [String]
 
-    public let extraSwiftCFlags: [String]
+    public private(set) var extraSwiftCFlags: [String] = []
```

Comment:
I'd rather not do this since we're already setting this in init.

Reply:
Okay, I'll just have some of the initialization duplicated then (since the problem is that we want additional flags in some cases and not others).